cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -742,10 +776,15 @@ def _add_default_instance_role(self, cleanup_policy_statements, build_tags):
                             account="",
                             resource="image",
                             resource_name="*",
-                        )
+                        ),
                     ],
                     actions=["ec2:CreateTags", "ec2:ModifyImageAttribute"],
-                )
+                ),
+                iam.PolicyStatement(
+                    effect=iam.Effect.ALLOW,
+                    resources=["*"],
+                    actions=["ec2:DescribeRegions"],
```

Comment:
I'm still not convinced to add this policy. While `CreateTags` and `ModifyImageAttribute` can be omitted while specifying a custom instance role, at the cost of not having all the tags and decription added into the AMI, this `DescribeRegions` will be required.
I'm in favour of removing the describe-regions test for now.

Reply:
removed